### ISSUE TITLE
The term Stop-DurableTaskTimer is not recognized

### ISSUE THREAD

**Issue by 4c74356b41** [Azure/azure-functions-powershell-worker#673]
#### Repro steps

Provide the steps required to reproduce the problem:

1. Create a powershell function with the following parameters
![image](https://user-images.githubusercontent.com/16194643/129448733-fef8d7e3-2d6a-4ee9-92f9-f46f2efbf156.png)
2. add an durable activity function with following code:
Sample from the [docs](https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-timers?tabs=powershell):

```
param($Context)
$expiryTime =  New-TimeSpan -Seconds 30

$activityTask = Invoke-DurableActivity -FunctionName 'GetQuote'-NoWait
$timerTask = Start-DurableTimer -Duration $expiryTime -NoWait
$winner = Wait-DurableTask -Task @($activityTask, $timerTask) -Any

if ($winner -eq $activityTask) {
    Stop-DurableTaskTimer -Task $timerTask
    return $True
}
else {
    return $False
}
```

#### Expected behavior

durable timer function is stopped from the orchestrator

#### Actual behavior

throws: `the term Stop-DurableTaskTimer is not recognized`

#### Known workarounds

None

#### Related information 

https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-timers?tabs=powershell

**Comment by AnatoliB** [comment]
This is a typo in the doc: the actual command name is `Stop-DurableTimerTask`.

@stefanushinardi We need to fix https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-timers?tabs=powershell

**Comment by 4c74356b41** [comment]
hey, thats charming. how about a page that specifies all the available cmdlets and their contracts? how am I supposed to figure that out without raising an issue?

**Comment by 4c74356b41** [comment]
oh sorry, while we are at it, is there a way to cancel an activity?

```
param($Context)
$expiryTime =  New-TimeSpan -Seconds 30

$activityTask = Invoke-DurableActivity -FunctionName 'GetQuote'-NoWait
$timerTask = Start-DurableTimer -Duration $expiryTime -NoWait
$winner = Wait-DurableTask -Task @($activityTask, $timerTask) -Any

if ($winner -eq $activityTask) {
    Stop-DurableTimerTask -Task $timerTask
    return $True
}
else {
    Stop-DurableActivity -Activity $activityTask # < this bad boy
    return $False
}
```

**Comment by AnatoliB** [comment]
@4c74356b41 Today, the primary way to discover the cmdlets and their contracts is via concepts documentation and tutorials, starting from use cases instead of a list of commands. But I see what you mean, we'll add a reference as well.

The framework does not provide any way to cancel an activity that already started. Depending on what this activity is doing, you may be able to do something to cancel it, for example, by making the activity function check for an external signal of some kind (e.g. an Azure Queue message) and then exit. However, the details depend very much on the nature of the activity, and the framework does not provide any generic way.

**Comment by 4c74356b41** [comment]
@AnatoliB cant you just publish a module to psgallery and link it? that would be more than enough to explore it

**Comment by AnatoliB** [comment]
@4c74356b41 This is the plan, but this is not a trivial thing to do right now: these commands are currently tightly coupled with the worker internals, so there is no standalone module that can be loaded into a regular (not hosted by the worker) PS session. This coupling will have to be resolved, or we'll use #577 as the last resort.

**Comment by AnatoliB** [comment]
The typo in the docs has been fixed, so I'm closing this. For other topics, please create separate issues.
